Peak Element

```diff
@@ -0,0 +1,18 @@
+# Time Complexity: O(log(n))
```

Comment:
Please explain what 'n' is

Reply:
'n' is the size of the given input, for suppose the given input has 'n' number of elements in it, size of the input is said to be 'n'.